atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -20,10 +23,35 @@
 from atomic_reactor.plugins.pre_add_filesystem import AddFilesystemPlugin
 from atomic_reactor.plugins.pre_check_and_set_rebuild import is_rebuild
 from atomic_reactor.constants import PLUGIN_KOJI_IMPORT_PLUGIN_KEY
-from atomic_reactor.util import (get_build_json, get_preferred_label, df_parser)
+from atomic_reactor.util import (get_build_json, get_preferred_label, df_parser, get_checksums)
 from atomic_reactor.koji_util import create_koji_session
 from osbs.conf import Configuration
 from osbs.api import OSBS
+from osbs.exceptions import OsbsException
+
+LogEntry = namedtuple('LogEntry', ['platform', 'line'])
+Output = namedtuple('Output', ['file', 'metadata'])
+
+
+class KojiUploadLogger(object):
```

Comment:
Yeah, was thinking that was the way to go as well, since it's 100% copy-n-paste, just wasn't sure if it was the right thing to copy-n-paste yet. I'll go ahead and make that change.